README.md

```diff
@@ -76,6 +76,21 @@ make man
  - **https**: `https://github.com/osbuild/osbuild-composer.git`
  - **ssh**:   `git@github.com:osbuild/osbuild-composer.git`
 
+### Pull request gating
+
+Each pull request against `osbuild-composer` starts a series of automated
+tests. Tests run via GitHub Actions, Travis CI, and Jenkins. Each push to
+the pull request will launch theses tests automatically.
+
+Jenkins only tests pull requests from members of the `osbuild` organization in
+GitHub. A member of the `osbuild` organization must say `ok to test` in a pull
+request comment to approve testing. Anyone can ask for testing to run by
+saying the bot's favorite word, `schutzbot`, in a pull request comment.
```

Comment:
Hm I'm not sure I understand when to say `schutzbot`. To re-run the tests?

Reply:
So the `ok to test` by an osbuild member is a signal to Jenkins that the PR has been approved, but it doesn't start the test itself. Typing `schutzbot` will cause Jenkins to run tests immediately. If you type it while a test is running, Jenkins will abort that test and start over with a new test.